test/unit/lib/plugins/aws/package/compile/events/cloudFront.test.js

```diff
@@ -2113,6 +2147,18 @@ describe('test/unit/lib/plugins/aws/package/compile/events/cloudFront.test.js',
       expect(getAssociatedCacheBehavior('managedPolicy').CachePolicyId).to.eq(cachePolicyId);
     });
 
+    it('Should attach a cache policy to a cloudfront behavior when specified by id in alternative way of lambda config', () => {
+      expect(getAssociatedCacheBehavior('managedPolicyInAlternativeWay').CachePolicyId).to.eq(
+        cachePolicyId
+      );
+    });
+
+    it('Should attach a cache policy specified in major way to a cloudfront behavior when specified by id in both ways of lambda config', () => {
```

Comment:
Let's point out directly how it was set instead of using ambiguous `major way` statement